DependencyInjection/SonataBlockExtension.php

```diff
@@ -72,7 +72,9 @@ public function load(array $configs, ContainerBuilder $container)
         $this->configureProfiler($container, $config);
         $this->configureException($container, $config);
         $this->configureMenus($container, $config);
-        $this->configureClassesToCompile();
+        if (\PHP_VERSION_ID < 70000) {
```

Comment:
Yeah plus it's less error prone, and it's still easy to ready with such a constant.